From 5361ed8259c9db5cf306c7cf5194af06732cb130 Mon Sep 17 00:00:00 2001
From: Amir Hammad <amir.hammad@streamunlimited.com>
Date: Fri, 11 Dec 2015 15:46:03 +0100
Subject: [PATCH 1/3] Fix for device 0x442: System memory start address

RM0091(rev.8.): page 50: STM32F0xx memory boundary addresses (continued)
also RM0360 (rev.3) : page 39: STM32F0x0 memory boundary addresses (continued)

0x1FFFC800 -> 0x1FFFD800

Signed-off-by: Amir Hammad <amir.hammad@streamunlimited.com>
---
 dev_table.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dev_table.c b/dev_table.c
index f4ada72..50fb64e 100755
--- a/dev_table.c
+++ b/dev_table.c
@@ -59,7 +59,7 @@ const stm32_dev_t devices[] = {
 	/* ID   "name"                              SRAM-address-range      FLASH-address-range    PPS  PSize   Option-byte-addr-range  System-mem-addr-range   Flags */
 	/* F0 */
 	{0x440, "STM32F030x8/F05xxx"              , 0x20000800, 0x20002000, 0x08000000, 0x08010000,  4, p_1k  , 0x1FFFF800, 0x1FFFF80F, 0x1FFFEC00, 0x1FFFF800, 0},
-	{0x442, "STM32F030xC/F09xxx"              , 0x20001800, 0x20008000, 0x08000000, 0x08040000,  2, p_2k  , 0x1FFFF800, 0x1FFFF80F, 0x1FFFC800, 0x1FFFF800, F_OBLL},
+	{0x442, "STM32F030xC/F09xxx"              , 0x20001800, 0x20008000, 0x08000000, 0x08040000,  2, p_2k  , 0x1FFFF800, 0x1FFFF80F, 0x1FFFD800, 0x1FFFF800, F_OBLL},
 	{0x444, "STM32F03xx4/6"                   , 0x20000800, 0x20001000, 0x08000000, 0x08008000,  4, p_1k  , 0x1FFFF800, 0x1FFFF80F, 0x1FFFEC00, 0x1FFFF800, 0},
 	{0x445, "STM32F04xxx/F070x6"              , 0x20001800, 0x20001800, 0x08000000, 0x08008000,  4, p_1k  , 0x1FFFF800, 0x1FFFF80F, 0x1FFFC400, 0x1FFFF800, 0},
 	{0x448, "STM32F070xB/F071xx/F72xx"        , 0x20001800, 0x20004000, 0x08000000, 0x08020000,  2, p_2k  , 0x1FFFF800, 0x1FFFF80F, 0x1FFFC800, 0x1FFFF800, 0},
-- 
2.9.2

